odoo/models.py

```diff
@@ -3075,7 +3075,7 @@ def get_field_translations(self, field_name, langs=None):
         :param str field_name: field name
         :param list langs: languages
 
-        :return dict translations: [(lang, val_en, val_lang)]
+        :return dict translations: [{lang: lang, source: val_en, value: val_lang)]
```

Comment:
It returns a list of dicts
```suggestion
        :return: list of dicts like [{"lang": lang, "source": source_term, "value": value_term}]
```